streams/src/main/java/org/apache/kafka/streams/state/internals/metrics/RocksDBMetrics.java

```diff
@@ -380,7 +379,7 @@ public static Sensor numberOfOpenFilesSensor(final StreamsMetricsImpl streamsMet
     public static Sensor numberOfFileErrorsSensor(final StreamsMetricsImpl streamsMetrics,
                                                   final RocksDBMetricContext metricContext) {
         final Sensor sensor = createSensor(streamsMetrics, metricContext, NUMBER_OF_FILE_ERRORS);
-        addSumMetricToSensor(
+        addValueMetricToSensor(
```

Comment:
Ah good catch.

Reply:
Actually should the previous sensors like `bytesReadDuringCompactionSensor` not be a `addRateOfSumMetricToSensor` metric than a rate? My understanding is that the values read from rocksDB is already sum, so we probably only cares about avg / max?

Reply:
The statistics ticker that we read from RocksDB is a monotonically increasing sum of bytes that we reset after each read. I chose a rate because it should give an impression about the IO that is produced by compactions.